modules/avoindata-drupal-theme/config/install/core.entity_view_display.node.page.default.yml

```diff
@@ -0,0 +1,50 @@
+langcode: fi
+status: true
+dependencies:
+  config:
+    - field.field.node.page.body
+    - field.field.node.page.field_basic_page_comments
+    - field.field.node.page.field_image
+    - node.type.page
+  module:
+    - disqus
+    - image
+    - text
+    - user
+_core:
+  default_config_hash: g1S3_GLaxq4l3I9RIca5Mlz02MxI2KmOquZpHw59akM
```

Comment:
```suggestion
```
I'm not a fan of including these hashes or uuids. If you change something manually in the config the hash should probably be updated too but no-one is going to do that.